.circleci/config.yml

```diff
@@ -66,24 +66,114 @@ jobs:
         environment:
           TOXENV: pypy3-core
-  py35-backends:
+  py35-backends-coincurve7:
     <<: *common
     docker:
       - image: circleci/python:3.5
         environment:
-          TOXENV: py35-backends
-  py36-backends:
+          TOXENV: py35-backends-coincurve7
+  py36-backends-coincurve7:
     <<: *common
     docker:
       - image: circleci/python:3.6
         environment:
-          TOXENV: py36-backends
-  py37-backends:
+          TOXENV: py36-backends-coincurve7
+  py37-backends-coincurve7:
     <<: *common
     docker:
       - image: circleci/python:3.7
         environment:
-          TOXENV: py37-backends
+          TOXENV: py37-backends-coincurve7
+  py35-backends-coincurve8:
+    <<: *common
+    docker:
+      - image: circleci/python:3.5
+        environment:
+          TOXENV: py35-backends-coincurve8
+  py36-backends-coincurve8:
+    <<: *common
+    docker:
+      - image: circleci/python:3.6
+        environment:
+          TOXENV: py36-backends-coincurve8
+  py37-backends-coincurve8:
+    <<: *common
+    docker:
+      - image: circleci/python:3.7
+        environment:
+          TOXENV: py37-backends-coincurve8
+  py35-backends-coincurve9:
+    <<: *common
+    docker:
+      - image: circleci/python:3.5
+        environment:
+          TOXENV: py35-backends-coincurve9
+  py36-backends-coincurve9:
+    <<: *common
+    docker:
+      - image: circleci/python:3.6
+        environment:
+          TOXENV: py36-backends-coincurve9
+  py37-backends-coincurve9:
+    <<: *common
+    docker:
+      - image: circleci/python:3.7
+        environment:
+          TOXENV: py37-backends-coincurve9
+  py35-backends-coincurve10:
+    <<: *common
+    docker:
+      - image: circleci/python:3.5
+        environment:
+          TOXENV: py35-backends-coincurve10
+  py36-backends-coincurve10:
+    <<: *common
+    docker:
+      - image: circleci/python:3.6
+        environment:
+          TOXENV: py36-backends-coincurve10
+  py37-backends-coincurve10:
+    <<: *common
+    docker:
+      - image: circleci/python:3.7
+        environment:
+          TOXENV: py37-backends-coincurve10
+  py35-backends-coincurve11:
+    <<: *common
+    docker:
+      - image: circleci/python:3.5
+        environment:
+          TOXENV: py35-backends-coincurve11
+  py36-backends-coincurve11:
+    <<: *common
+    docker:
+      - image: circleci/python:3.6
+        environment:
+          TOXENV: py36-backends-coincurve11
+  py37-backends-coincurve11:
+    <<: *common
+    docker:
+      - image: circleci/python:3.7
+        environment:
+          TOXENV: py37-backends-coincurve11
+  py35-backends-coincurve12:
+    <<: *common
+    docker:
+      - image: circleci/python:3.5
+        environment:
+          TOXENV: py35-backends-coincurve12
+  py36-backends-coincurve12:
+    <<: *common
+    docker:
+      - image: circleci/python:3.6
+        environment:
+          TOXENV: py36-backends-coincurve12
+  py37-backends-coincurve12:
```

Comment:
curious if there is a way to autogenerate this stuff....  implying no action on this PR tho

Reply:
No good way that I know of.